dotcom-rendering/src/model/window-guardian.ts

```diff
@@ -119,80 +102,43 @@ export const makeWindowGuardian = (
 			reportError: (error: Error, feature: string) => void;
 		};
 	};
-	GAData: GADataType;
+	GAData?: GADataType;
 } => {
 	return {
-		config: makeWindowGuardianConfig(CAPIArticle),
-		polyfilled: false,
-		adBlockers: {
-			active: undefined,
-			onDetect: [],
-		},
-		modules: {
-			sentry: {
-				reportError: () => null,
+		config: {
+			// This indicates to the client side code that we are running a dotcom-rendering rendered page.
+			isDotcomRendering: true,
+			isDev: process.env.NODE_ENV !== 'production',
+			stage: stage,
+			frontendAssetsFullURL: frontendAssetsFullURL,
+			page: Object.assign(unknownConfig, {
+				dcrCouldRender: true,
+				contentType: contentType ?? '',
+				edition: editionId,
+				revisionNumber,
+				dcrSentryDsn:
+					'https://1937ab71c8804b2b8438178dfdd6468f@sentry.io/1377847',
+				sentryPublicApiKey,
+				sentryHost,
+				keywordIds,
+				dfpAccountId,
+				adUnit,
+				showRelatedContent: true,
+				ajaxUrl,
+				shouldHideReaderRevenue: shouldHideReaderRevenue ?? false,
+				isPaidContent: isPaidContent ?? false,
+				brazeApiKey,
+			}),
+			libs: {
+				googletag: googletagUrl,
+			},
+			switches: switches,
+			tests: abTests,
+			ophan: {
+				pageViewId: '',
+				browserId: '',
```

Comment:
I know this was already set up that way, but what is the benefit of this? Should we not set it to `null` instead? Is this because we have typed these as `string` but they can only be determined client-side?

Not quite one for this PR to fix, but worth thinking about.

Reply:
I think you're right. I'd be much happier if these were `null` or `undefined` as that better represents their true state while they wait to get set by the script in the head of the page.

My guess is the reason we've not done this is we probably have contracts with external codebases reading this property which perhaps expect a string

Reply:
> Is this because we have typed these as `string` but they can only be determined client-side?

Yes, there's a piece of javascript that gets executed super early in the head of the page which sets `pageViewId` and `browserId` is set by Fastly and there's an init script which runs and updates this window property with it.

Arguably, we don't really need to use the window object to set `browserId`, you could just read the cookie directly 🤔